### ISSUE TITLE
Opening Poe Trade but cant find overlay or pressing shift + space doesnt work.

### ISSUE THREAD

**Issue by haruald9** [SnosMe/awakened-poe-trade#854]
Tried uninstalling and reinstalling. doesnt work. need help

**Comment by DZweek** [comment]
same issue. ctrl + D won't open trade prices overview most of the time and shift + space does nothing

**Comment by levotrea** [comment]
I've got the same issue, tested on 3.20.10004 and 3.20.10005. A way I found to make the app work in the game is to open it in the browser (via the option in the tray icon), open the settings by clicking on the cog, and clicking on save (no need to change anything in the settings).

This magically launches APT in the game with every hotkeys working.

**Comment by djhazlett** [comment]
It has to be something with how it detects Path running. If I exit apt, then reopen, it works again.

**Comment by SnosMe** [comment]
Any of you using VPN when launching Awakened?

**Comment by norecha** [comment]
Same issue [3.20.10004](https://github.com/SnosMe/awakened-poe-trade/releases/tag/v3.20.10004). No vpn.

Started happening after the update. I think it happens when I put pc to sleep and come back again, open poe, and it stops working. ctrl+d turns poe cursor into windows cursor but the ui does not pop up.

**Comment by djhazlett** [comment]
In my case, it does happen after sleep. No VPN involved. When I start POE I do not see the Awakened Poe Trade message, and nothing works. It's like it doesn't realize POE is running. Windows 11, every update installed. It started with this current version.

**Comment by levotrea** [comment]
> Any of you using VPN when launching Awakened?

I am indeed using a VPN, and turning it off before launching APT seems to work. Although the issue still appeared recently as I've been using a VPN while playing for more than a year and never had a problem with APT before.

**Comment by SnosMe** [comment]
Check the latest release whether it fixed the issue for you.

**Comment by levotrea** [comment]
It didn't solve the issue of APT not working when launched using a VPN. However, I tested with version 3.20.10003 and it works fine, so the issue has probably been introduced in 3.20.10004. In any case thanks for the work.

**Comment by SnosMe** [comment]
@levotrea try alt-tabbing between tab and some other window. Is more such errors appear?
For some reason, your VPN doesn't allow WebSockets traffic, even tho it's flowing completely local.

**Comment by levotrea** [comment]
I tried switching to other tabs in the browser, alt-tabbing to other windows, but no more errors were appearing.
However I also tried using Firefox instead of Opera GX, and this time I didn't have any error in the console and APT launched in game as soon as I opened it in the browser, so this particular error might be caused by Opera GX. But I still need to open APT in the browser for it to launch in game. I'll look into my VPN's config to see if it blocks any traffic locally.
Thanks for the help!

**Comment by djhazlett** [comment]
Update fixed my issue, in regard to after sleep, Awakened wouldn't see POE launch.

**Comment by verdantgrapes** [comment]
I also have a similar issue. It used to work just fine for me but I'm not sure at what point I started getting this [appended image] when I right-clicked on the tray icon. No VPN, restarted everything, uninstalled and cleared all Awakened files before reinstalling but I still ended up with these options when right-clicking. No option to view logs.

Same issue as the rest, no overlay in-game, shift+space doesn't do anything (but hotkeys for chat commands work) and price-checking interface only works in-browser.

![reducedoptions_poeawakened](https://user-images.githubusercontent.com/123139397/213614927-749ca6c7-135d-43dd-979d-94a863128409.png)

**Comment by verdantgrapes** [comment]
> I also have a similar issue. It used to work just fine for me but I'm not sure at what point I started getting this [appended image] when I right-clicked on the tray icon. No VPN, restarted everything, uninstalled and cleared all Awakened files before reinstalling but I still ended up with these options when right-clicking. No option to view logs.
> 
> Same issue as the rest, no overlay in-game, shift+space doesn't do anything (but hotkeys for chat commands work) and price-checking interface only works in-browser.
> 
> ![reducedoptions_poeawakened](https://user-images.githubusercontent.com/123139397/213614927-749ca6c7-135d-43dd-979d-94a863128409.png)

Yoooo, craziest thing. I updated my drivers and somewhere along the way, things started working again. The in-game overlay works just fine now.

**Comment by SnosMe** [comment]
I believe I fixed the issue, and it's not related to VPN. If I'm wrong, you can now add `--listen=0.0.0.0` option to the shortcut.

**Comment by micheilsgrey** [comment]
Maybe this will be helpful for some cases/users. I did get fixed 'hard way' my problem with this issue by simply deleting my antivirus (i wanted to do this anyways) ESET Internet Security. After deleting this antivirus i did get access to overlay and everything start working correctly. Looks like the firewall was blocking access to this app or maybe something else, I'm too dumb to understand what exactly causes this problem. I tried simply disabling the firewall in antivirus but still get the same problem. On ly deleting antivirus helped. And i found that this is not the first time someone has an issue with localhost and this antivirus.
